ambrs/ppe.py

```diff
@@ -198,10 +198,20 @@ def lhs(specification: EnsembleSpecification,
                          for f, mass_fraction in enumerate(mode.mass_fractions)]),
                 ) for m, mode in enumerate(specification.size.modes)]),
         )
-        # normalize mass fractions
+
+        # FIXME: laura's janky fix to set some mass fractions to zero
+        for mode in size.modes:
+            for qq in range(len(mode.mass_fractions)):
+                print(mode.mass_fractions[qq])
+                for ii in range(len(mode.mass_fractions[qq])):
+                    if np.isnan(mode.mass_fractions[qq][ii]):
```

Comment:
Are we seeing NaNs in mass fractions after creating an ensemble?

Reply:
NaNs started popping up when I attempted to zero-out some species. As a quick fix, I sampled from the uniform distribution from (0,0), which created NaNs. These led to all species in that mode going to NaNs after I normalized.

I think the best solution would be to implement an approach to set values to a fixed value across ensembles. This a feature we'd like for other reasons (e.g., running an ensemble all at the same RH, etc.). I'm sure there's an easier way to do this than sample a uniform distribution.